frontend/docs/types/branded-types.md

```
@@ -0,0 +1,86 @@
+```typescript
+declare const __brand: unique symbol;
+type Brand<B> = { [__brand]: B };
+
+export type Branded<T, B> = T & Brand<B>;
+```
+
+## 도메인별 ID 타입 위치
+
+도메인 ID 타입은 해당 도메인 파일에 정의한다. 타입 수가 늘어날수록 한 파일에 모으면 도메인 간 결합도가 높아지므로 아래 구조를 따른다.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)